packages/widgets-mitosis/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "widgets-mitosis",
```

Comment:
I wonder if there's a better name than `packages/widgets-mitosis` for this. Ideally `mitosis` wouldn't be mentioned as it's an implementation detail. 

The reason this is important this that once it exists, and we start to link to it for our users, it will be impossible to change the name in the future 😄. Let's check with our team for ideas.